api/analyzers/c/analyzer.py

```
@@ -1,476 +1,531 @@
+            return
+
+        splitted = os.path.splitext(normalized_path)
+        if len(splitted) < 2:
+            logger.warning("Include path has no extension: %s", included_file_path)
+            return
+
+        # Create file entity for the included file
+        path = os.path.dirname(normalized_path)
+        name = os.path.basename(normalized_path)
+        ext = splitted[1]
+        included_file = File(path, name, ext)
```

Ruff (0.15.6)

[error] `File` may be undefined, or defined from star imports

(F405)